docs/source/processing-levels.md

```diff
@@ -0,0 +1,61 @@
+# Proposed Echosounder Data Proccesing Levels (DRAFT)
+
+The decades-long experience from the satellite remote sensing community has shown that a set of robust and well-articulated definitions of "data processing levels" [1]–[5] can lead directly to broad and highly productive use of the data. Processing level designations also provide important context for data interpretation [6]. However, no such community agreement exists for active acoustic data, and the ambiguity associated with the interoperability and inter-comparability of processed sonar data products has hindered efficient collaboration and integrative use of the rapidly accumulated data archive across research institutions and agencies.
+
+The `echopype` team is developing a clearly defined progression of data processing levels for active ocean sonar data, leveraging the accumulated experience from remote sensing and large-scale, long-term ocean and ecological observatories [7]–[10]. Data processing functions in `echopype` are clearly associated with specific Processing Level inputs and outputs, and when appropriate, will generate a `processing_level` dataset global attribute with entries such as "Level 1A", "Level 2B", "Level 4", etc.
+
+
+## Processing Levels and Sub-levels
+
+### Level 0 (L0)
+
+**Description:**  Raw data in vendor sensor format.
+
+- raw binary files. Associated metadata may be found in external files.
+
+### Level 1 (L1)
+
+**Description:** Raw data packaged with ancillary information and converted and standardized to an open convention and standard data formats. May be distributed as sets of individual converted files as originally segmented into arbitrary time ranges during sensor file creation or compiled into larger granules corresponding to logical deployment intervals.
+
+- **L1A**: Raw L0 data converted to a standardized, open format with geographic coordinates (latitude & longitude) processed and included. Includes other ancillary information extracted from sensor-generated L0 data or other external sources. May include environmental information such as temperature, salinity and pressure. Use of the SONAR-netcDF4 v1 convention is strongly recommended.
```

Comment:
In "Raw L0 data converted to a standardized, open format with geographic coordinates (latitude & longitude) processed and included." it is not clear what "processed" means.

Reply:
I've removed "processed"